types/wlr_virtual_pointer_v1.c

```diff
@@ -0,0 +1,284 @@
+
+#include <assert.h>
+#include <stdlib.h>
+#include <wlr/types/wlr_seat.h>
+#include <wlr/types/wlr_virtual_pointer_v1.h>
+#include <wlr/types/wlr_pointer.h>
+#include <wlr/util/log.h>
+#include "util/signal.h"
+#include "wlr-virtual-pointer-unstable-v1-protocol.h"
+
+static void input_device_destroy(struct wlr_input_device *dev) {
+}
```

Comment:
Needs `/* This space deliberately left blank */` if deliberate, or an implementation if not